src/auth/dto/authcreds.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsString, IsNotEmpty, IsEmail } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class AuthCredsDto {
+  @IsString()
+  @ApiProperty({ description: 'The email of the user' })
+  @IsEmail()
+  @IsNotEmpty()
+  email: string;
+
+  @IsString()
+  @ApiProperty({ description: 'The password of the user' })
+  @IsNotEmpty()
```

Comment:
PullProofAI

It's recommended to include a minimum and maximum length validation for the password field to enhance security.
```diff
+   @MinLength(8)
+   @MaxLength(20)
```

### Summary
The suggested changes aim to improve data validation by enforcing constraints on the email and password fields in the `AuthCredsDto` class.


<!-- This is an auto-generated comment by PullProofAI -->